content/millennium/r4/medications/medication_request.md

```diff
@@ -0,0 +1,144 @@
+---
+title: MedicationRequest | R4 API
+---
+
+# MedicationRequest
+
+* TOC
+{:toc}
+
+## Overview
+
+The Medication Request resource provides orders for all medications along with administration instructions for a
+patient in both the inpatient and outpatient setting (orders/prescriptions filled by a pharmacy and discharge
+medication orders). This resource also includes a patients historical or documented home medications reported
+by the patient, significant other or another provider.   
+
+If the Medication Request represents a prescription (something the patient takes at home),
+the start, stop, and other data may not be a representation of when the medication was taken.
+For example, the system may not know if the patient ever filled or took the prescribed medication,
+or when the prescription was filled. Documented historical/past/home medications are commonly captured
+when taking the patient’s medical history.
+
+The following fields are returned if valued:
+
+* [MedicationRequest id]( https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.status){:target="_blank"}
+* [Status Reason](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.statusReason){:target="_blank"}
+* [Intent](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.intent){:target="_blank"}
+* [Category](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.category){:target="_blank"}
+* [Priority](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.priority){:target="_blank"}
+* [Reported Boolean](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.reported_x_){:target="_blank"}
+* [Medication](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.medication_x_){:target="_blank"}
```

Comment:
if we write Reported Boolean I suppose we should write Medication Reference/CodeableConcept. or is it the case that we return both references and ccs?

Reply:
We can return either a medicationCodeableConcept or a medicationReference.